tests/draw_test.py

```diff
@@ -162,5 +162,10 @@ def test_draw_biome(self):
         draw_biome(w, target)
         self._assert_img_equal("biome_28070", target)
 
+    def test_draw_scatter_plot(self):
```

Comment:
It's a coverage test. It tests pretty much all of the new code (since the world being used contains all temperature and humidity ranges).